[2025-07-18T04:00:02.150+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_analise_desempenho_revendas.executar_query_gnio scheduled__2025-07-17T04:00:00+00:00 [queued]>
[2025-07-18T04:00:02.165+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_analise_desempenho_revendas.executar_query_gnio scheduled__2025-07-17T04:00:00+00:00 [queued]>
[2025-07-18T04:00:02.165+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-07-18T04:00:02.212+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): executar_query_gnio> on 2025-07-17 04:00:00+00:00
[2025-07-18T04:00:02.223+0000] {standard_task_runner.py:60} INFO - Started process 1413223 to run task
[2025-07-18T04:00:02.229+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'DAG_analise_desempenho_revendas', 'executar_query_gnio', 'scheduled__2025-07-17T04:00:00+00:00', '--job-id', '555', '--raw', '--subdir', 'DAGS_FOLDER/DagAnaliseDesempenhoRevendas.py', '--cfg-path', '/tmp/tmp20ad0en0']
[2025-07-18T04:00:02.235+0000] {standard_task_runner.py:88} INFO - Job 555: Subtask executar_query_gnio
[2025-07-18T04:00:02.669+0000] {task_command.py:423} INFO - Running <TaskInstance: DAG_analise_desempenho_revendas.executar_query_gnio scheduled__2025-07-17T04:00:00+00:00 [running]> on host 99f5714845f7
[2025-07-18T04:00:02.975+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_analise_desempenho_revendas' AIRFLOW_CTX_TASK_ID='executar_query_gnio' AIRFLOW_CTX_EXECUTION_DATE='2025-07-17T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-17T04:00:00+00:00'
[2025-07-18T04:00:03.007+0000] {base.py:83} INFO - Using connection ID 'gnio' for task execution.
[2025-07-18T04:00:04.154+0000] {warnings.py:109} WARNING - /opt/***/dags/DAG_atualizacao_desempenho_revendas/AnaliseDesempenhoRevendas.py:16: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(sql, connection_database)

[2025-07-18T04:00:06.454+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-18T04:00:06.478+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=DAG_analise_desempenho_revendas, task_id=executar_query_gnio, execution_date=20250717T040000, start_date=20250718T040002, end_date=20250718T040006
[2025-07-18T04:00:06.578+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-18T04:00:06.871+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
